kiwix-build.py

```diff
@@ -399,7 +400,14 @@ def download(self, what, where=None):
             if what.sha256 == get_sha256(file_path):
                 raise SkipCommand()
             os.remove(file_path)
-        urllib.request.urlretrieve(file_url, file_path)
+        if options.no_cert_check == True:
+            context = ssl.create_default_context()
+            context.check_hostname = False
+            context.verify_mode = ssl.CERT_NONE
+            with urllib.request.urlopen(file_url, context=context) as resource, open(file_path, 'wb') as file:
+                file.write(resource.read())
```

Comment:
> Why don't we keep both for now?

Because I don't want to comeback on this point latter :) The less code branches there is, the better it is.

This should not be to difficult to implement, a small read in a loop should be enough.
I've check `urlretrieve` and this is globally what it does (https://github.com/python/cpython/blob/master/Lib/urllib/request.py#L229). It reads small chunk of 8Ko in a `while` loop.